### DISCUSSION TITLE
Ordem de atributos no Jade

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#285] [Perguntas e Respostas]
Olá, pessoal.

Há algum dias eu venho procurando alguma opção que possa configurar a ordem de atributos na hora de compilar. Tendo o [Code Guide](http://codeguide.co/#html-attribute-order) como referência, seria muito interessante adotar esse padrão.

Alguém sabe me dizer se isso é possível? Obrigado :)

**Comment by felipefialho** [comment]
Mas você realmente vê necessidade?

Porque é um arquivo compilado. Eu acho que faria mais sentido organizar a ordem no próprio Jade, que é onde você vai desenvolver.

**Comment by ghost** [comment]
@LFeh, vejo. Pois muita das vezes eu preciso fazer o html estático para depois integrar com o WordPress.

**Comment by ghost** [comment]
Eu achei duas issues que me parecem com esta minha dúvida. Inclusive um dos comentários ele passa como referência o link do Code Guide.
- https://github.com/pugjs/pug/issues/1914
- https://github.com/pugjs/pug/issues/1793
